namespaces/live-1.cloud-platform.service.justice.gov.uk/cronjobber/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
I think we should have this as a separate job in concourse (building the image). Right now I'm assuming the intention is to build manually whenever we need to?

Reply:
kube-system is a small change so sure, will check Concourse too and amend